examples/with-imported-react.js

```diff
@@ -1,6 +1,8 @@
 import React from 'react'
 
 export default class Component {
+  static displayName() {}
```

Comment:
Because i've had a error about missing display name property in definition class name Component.